build/Data/MODELS.TXT

```diff
@@ -68,3 +69,4 @@ know	Crab		data/chars/crab/crab.txt
 know	Lobster		data/chars/Lobster/Lobster.txt
 load Golrio data/chars/golrio/golrio.txt
 load Washizuka data/chars/M_washizuka/M_washizuka.txt
+load	leo		    data/chars/leo/leo.txt
```

Comment:
@Neverlandse00  FYI, when adding anyone that isn't a player/character, then you used "know". If they are a character, then you use "load". Just an FYI. We can fix later but wanted to let you know that.